onnxruntime/python/tools/transformers/models/bart/utils/export_summarization_enc_dec_past.py

```diff
@@ -0,0 +1,265 @@
+import copy
+import os
+import time
+
+import torch
+from transformers import BartConfig, BartForConditionalGeneration, file_utils
+from utils import export_helper
+
+from onnxruntime import InferenceSession, SessionOptions
+
+
+def decoder_config_update(config: BartConfig):
+    """
+    Add parameters into decoder config to help expoerting. These parameter
+    are later consumed with control flow and assertion
+
+    Args:
+        config: BART config
+    """
+
+    new_attributes = {
+        "is_decoder_exported": False,
+        "is_decoder_with_past_exported": False,
+        "during_export": False,
+        "expected_args": {"use_cache": True, "return_dict": True},
+        "expected_inputs": ["decoder_input_ids", "attention_mask", "encoder_outputs"],
+    }
+    config.update(new_attributes)
+    return config
+
+
+class DecoderWrapper(torch.nn.Module):
+    """main part of BART Decoder wrapper.
+
+    We split BART into three parts: Encoder plus initial part of Decodoer, main part of Decoder, and Beam Search.
```

Comment:
Decodoer => decoder. Also update the following line.

Reply:
Done